modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheContext.java

```diff
@@ -2348,6 +2353,16 @@ public AtomicReference<IgniteInternalFuture<Boolean>> lastRemoveAllJobFut() {
         return lastRmvAllJobFut;
     }
 
+    /** */
+    public DumpEntryChangeListener dumpListener() {
+        return dumpLsnr;
+    }
+
+    /** */
+    public void dumpListener(DumpEntryChangeListener dumpEntryChangeLsnr) {
+        this.dumpLsnr = dumpEntryChangeLsnr;
```

Comment:
Let's add assert that the current `dumpLsnr` is null.

Reply:
Fixed.